modules/ossm-security-mtls.adoc

```diff
@@ -61,3 +61,38 @@ spec:
     tls:
       mode: ISTIO_MUTUAL
 ----
+
+== Setting the minimum and maximum protocol versions
+
+If your environment has specific requirements for encrypted traffic in your service mesh, such as for GDPR, you can control the cryptographic functions that are allowed by setting the `spec.istio.global.tls.minProtocolversion` or `spec.istio.global.tls.maxProtocolversion` in your ServiceMeshControlPlane resource. Those values define the minimum and maximum TLS version in your control plane to use minimum or maximum level of TLS across all encrypted traffic in your mesh.
```

Comment:
- It may be better not to explicitly mention GDPR nor other security policies.
- We should stress that these are the default values for minimum/maximum.
- minProtocolVersion and maxProtocolVersion (capital V)
- Perhaps
"Those values, configured in your control plane resource, define the default minimum and maximum TLS version used by mesh components when communicating securely over TLS"